src/components/navbar/navBar.scss

```diff
@@ -0,0 +1,93 @@
+.navBar {
```

Comment:
it is more conventional write class names in kebab-case (nav-bar) than camelCase (navBar)